src/api/db/migrate/20170111114943_add_exclude_projects_to_configuration.rb

```diff
@@ -0,0 +1,17 @@
+class AddExcludeProjectsToConfiguration < ActiveRecord::Migration[5.0]
+  def self.up
+    add_column :configurations, :excluded_projects, :string
+    add_column :configurations, :excluded_projects_description, :string
+
+    # update database from file
+    old = CONFIG['global_write_through']
+    CONFIG['global_write_through'] = false
+    ::Configuration.first.update_from_options_yml
+    CONFIG['global_write_through'] = old
```

Comment:
We saw it here https://github.com/openSUSE/open-build-service/blob/master/src/api/db/migrate/20130702083665_options_yml_to_configuration.rb#L23 and thought that maybe is needed... but taking a look at it right now I don't have it clear. Maybe @adrianschroeter remember why it was needed then 😃